ossrelease/msgs/branch

```diff
@@ -0,0 +1 @@
+Just wanted to give you a heads up that we will branch _salt_version_ soon. The tag will follow a couple days afterwards, after all PRs are merged and automated tests are passing. Please email open-release-team@saltstack.com if you want to ensure your PR is merged for the upcoming release if it is not currently in that branch.
```

Comment:
Maybe this should match the other `live_soon` message and be more formal? I am thinking that a `Hi all,` at the beginning would be good. Maybe remove the `heads up` part and simplify it:

```
Hi all,

We will be branching _salt_version_ soon. The tag will follow ...
```

Since this is automated now, I think that would be good change to make.